core/src/test/java/dk/alexandra/fresco/framework/util/TestModularReduction.java

```diff
@@ -0,0 +1,34 @@
+package dk.alexandra.fresco.framework.util;
+
+import static org.junit.Assert.assertEquals;
+
+import java.math.BigInteger;
+
+import org.junit.Test;
+
+public class TestModularReduction {
```

Comment:
We should at a minimum test for:

* Each modulus we plan on using in SPDZ (i.e., those in ModulusFinder)
* Non-negative inputs smaller than the modulus
* Edge-cases: zero input, input equal to modulus

There are probably more edge cases, but I haven't looked through the implementation yet.

Reply:
I extended the tests (which caught an error :))